src/main/java/teammates/ui/template/CourseEditInstructorPanel.java

```diff
@@ -0,0 +1,189 @@
+package teammates.ui.template;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import teammates.common.datatransfer.InstructorAttributes;
+import teammates.common.util.Const;
+
+public class CourseEditInstructorPanel {
+    public int index;
+    public InstructorAttributes instructor;
+    public List<ElementTag> permissionInputGroup1;
+    public List<ElementTag> permissionInputGroup2;
+    public List<ElementTag> permissionInputGroup3;
```

Comment:
I don't know if I can find meaningful names for them. It looks like they are separated just for displaying purpose only :/